tests/test_features/test_pca.py

```diff
@@ -164,7 +164,7 @@ def test_biplot_3d(self):
         pca_array = visualizer.transform(self.dataset.X)
 
         # Image comparison tests
-        self.assert_images_similar(visualizer)
+        self.assert_images_similar(visualizer,tol=10)
 
         # Assert PCA transformation occurred successfully
         assert pca_array.shape == (self.dataset.X.shape[0], 3)
```

Comment:
We might want to add a test for alpha.  It could be similar to the alpha test found in residuals.py

Reply:
OK. I will construct a function for alpha test.